packages/core/src/shipping/strategies/braintree/braintree-accelerated-checkout-shipping-strategy.ts

```diff
@@ -0,0 +1,219 @@
+import {
+    BraintreeConnectAddress,
+    BraintreeConnectVaultedInstrument,
+    BraintreeIntegrationService,
+} from '@bigcommerce/checkout-sdk/braintree-utils';
+import { BrowserStorage } from '@bigcommerce/checkout-sdk/storage';
+
+import { AddressRequestBody } from '../../../address';
+import { BillingAddressActionCreator } from '../../../billing';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import {
+    InvalidArgumentError,
+    MissingDataError,
+    MissingDataErrorType,
+} from '../../../common/error/errors';
+import { CustomerAddress } from '../../../customer';
+import { Country } from '../../../geography';
+import { PaymentMethodActionCreator } from '../../../payment';
+import { PaymentProviderCustomerActionCreator } from '../../../payment-provider-customer';
+import { CardInstrument } from '../../../payment/instrument';
+import { UntrustedShippingCardVerificationType } from '../../../payment/instrument/instrument';
+import ConsignmentActionCreator from '../../consignment-action-creator';
+import { ShippingRequestOptions } from '../../shipping-request-options';
+import ShippingStrategy from '../shipping-strategy';
+
+import BraintreeAcceleratedCheckoutInitializeOptions from './braintree-accelerated-checkout-initialize-options';
+
+export default class BraintreeAcceleratedCheckoutShippingStrategy implements ShippingStrategy {
+    private browserStorage: BrowserStorage;
+
+    constructor(
+        private store: CheckoutStore,
+        private billingAddressActionCreator: BillingAddressActionCreator,
+        private consignmentActionCreator: ConsignmentActionCreator,
+        private paymentMethodActionCreator: PaymentMethodActionCreator,
+        private paymentProviderCustomerActionCreator: PaymentProviderCustomerActionCreator,
+        private braintreeIntegrationService: BraintreeIntegrationService,
+    ) {
+        this.browserStorage = new BrowserStorage('paypalConnect');
```

Comment:
```suggestion
        this._browserStorage = new BrowserStorage('paypalConnect');
```

Reply:
done

Reply:
DC corrected me this before. 😂
>We don't really use `_` as a convention for private members in checkout-js because it's not published as a library.
https://github.com/bigcommerce/checkout-js/pull/1392#discussion_r1275649501